Configuration/Skimming/python/PbPb_ZMuSkimMuonDPG_cff.py

```diff
@@ -0,0 +1,128 @@
+import FWCore.ParameterSet.Config as cms
+### HLT filter
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+from PhysicsTools.PatAlgos.producersLayer1.genericParticleProducer_cfi import patGenericParticles
+
+
+ZMuHLTFilter = copy.deepcopy(hltHighLevel)
+ZMuHLTFilter.throw = cms.bool(False)
+ZMuHLTFilter.HLTPaths = ["HLT_HIL3Mu*"]
+
+### Z -> MuMu candidates
+# Get muons of needed quality for Zs
+
+###create a track collection with generic kinematic cuts
+looseMuonsForZMuSkim = cms.EDFilter("TrackSelector",
+                             src = cms.InputTag("generalTracks"),
+                             cut = cms.string('pt > 10 &&  abs(eta)<2.4 &&  (charge!=0)'),
+                             filter = cms.bool(True)                                
+                             )
+
+
+
+###cloning the previous collection into a collection of candidates
+ConcretelooseMuonsForZMuSkim = cms.EDProducer("ConcreteChargedCandidateProducer",
+                                              src = cms.InputTag("looseMuonsForZMuSkim"),
+                                              particleType = cms.string("mu+")
+                                              )
+
+
+
+###create iso deposits
+tkIsoDepositTk = cms.EDProducer("CandIsoDepositProducer",
```

Comment:
@mandrenguyen this and the next module look carbon-copies of what already in tn ZMuSkim, wouldn't be better to differentiate the name appending also here a ForZMuSkim in order to avoid possible conflicts/inconsistencies/confusions?